godot-codegen/src/util.rs

```diff
@@ -268,6 +298,23 @@ fn to_rust_type_uncached(ty: &str, ctx: &mut Context) -> RustTy {
         }
     }
 
+    if ty.ends_with('*') {
+        // Pointer type; strip '*', see if const, and then resolve the
+        // inner type.
+        let mut ty = ty.to_owned();
+        ty.pop();
+        // 'const' should apply to the innermost pointer, if present.
+        let is_const = ty.starts_with("const ") && !ty.ends_with('*');
+        if is_const {
+            ty = ty.replace("const ", "");
+        }
+        let inner_type = to_rust_type(ty.trim(), ctx);
```

Comment:
Is the `trim()` really needed? If not, please remove it 🙂

Reply:
This one is necessary, unfortunately. :( The extension API json contains this gem.
```
"type": "const uint8_t **"
```
When they have a double pointer, they put a space between the stars and the type proper.

Reply:
I see. Can you add a comment that explains this reason? Thanks!

Reply:
:+1: